tests/ballerina-integration-test/src/test/resources/http2/http2services/03_http_2.0_to_http_1.0_fallback_test.bal

```diff
@@ -16,15 +16,42 @@
 
 import ballerina/http;
 
-endpoint http:Listener helloWorldEP {
+endpoint http:Listener serviceEndpointWithoutSSL {
     port: 9095,
     httpVersion: "2.0"
 };
 
+endpoint http:Listener serviceEndpointWithSSL {
+    port: 9096,
+    httpVersion: "2.0",
+    secureSocket: {
+        keyStore: {
+            path: "${ballerina.home}/bre/security/ballerinaKeystore.p12",
+            password: "ballerina"
+        }
+    }
+};
+
+@http:ServiceConfig {
+    basePath: "/hello"
+}
+service helloWorldWithoutSSL bind serviceEndpointWithoutSSL {
+
+    @http:ResourceConfig {
+        methods: ["GET"],
+        path: "/"
+    }
+    sayHelloGet(endpoint caller, http:Request req) {
+        http:Response res = new;
+        res.setTextPayload("Version: " + untaint req.httpVersion);
+        _ = caller->respond(res);
```

Comment:
We could straightaway set the payload to the respond()

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10951/commits/34db2e3f56d52290a4edc2aa1ef4f7575750d2b1